addons/binding/org.openhab.binding.chromecast/src/main/java/org/openhab/binding/chromecast/handler/ChromecastHandler.java

```diff
@@ -166,12 +169,25 @@ public void handleCommand(final ChannelUID channelUID, final Command command) {
             case ChromecastBindingConstants.CHANNEL_VOLUME:
                 handleVolume(command);
                 break;
+            case ChromecastBindingConstants.CHANNEL_PLAY_URI:
+                handlePlayUri(command);
+                break;
             default:
                 logger.debug("Received command {} for unknown channel: {}", command, channelUID);
                 break;
         }
     }
 
+    private void handlePlayUri(Command command) {
+        if (command instanceof StringType) {
+            if (command.equals("NULL")) {
```

Comment:
I tried both NULL and null on the REST API and they came through as StringType (rather than UnDefType.NULL which is a State). I can't see a way to compare a Command to a State/Type.  The command.toString().isEmpty() also comes back false for NULL and null.

So I'm not sure how best to change that condition.